src/index.ts

```diff
@@ -195,9 +186,6 @@ export class Tunnel extends EventEmitter {
 
 // Proxy authorization middleware for http tunnel.
 export const basicAuth: Middleware = (req, _cltSocket, _head, next) => {
-	const proxyAuth = req.headers['proxy-authorization'];
-	if (proxyAuth != null) {
-		req.auth = basicAuthParser(proxyAuth);
-	}
+	req.auth = basicAuthParser(req);
```

Comment:
Does this not need to look at the proxy auth anymore?

Reply:
whoops, nice catch.